package.json

```diff
@@ -32,10 +32,10 @@
     "clean": "backstage-cli package clean"
   },
   "dependencies": {
-    "@backstage/catalog-model": "^1.1.3",
-    "@backstage/core-components": "^0.12.0",
-    "@backstage/core-plugin-api": "^1.1.0",
-    "@backstage/plugin-catalog-react": "^1.2.1",
+    "@backstage/catalog-model": "^1.1.4",
+    "@backstage/core-components": "^0.12.2",
```

Comment:
https://backstage.github.io/upgrade-helper/?from=1.8.1&to=1.9.1 states that backstage v1.9.1 is using this package in version `^1.12.1`. Might be better to replicate this here.

Reply:
Same as in the case below